src/pages/faq.tsx

```diff
@@ -185,14 +136,16 @@ export default function Faq() {
       {/*Dropdown table*/}
       
       {/*Product Catalog button*/}
-      <button className="mt-[100px] h-[100px] w-[250px] bg-cerise rounded-[40px] border-cerise flex items-center justify-center px-[30px] flex-col">
-        <a href="/downloadables/Product_Catalog.pdf" download="Product_Catalog" className="text-white text-[22px] text-center">{t.faq.download}</a>
-        <div className="flex items-center">
-          <span className="text-white text-[20px]">{t.faq.productCatalog}</span>
-        </div>
-      </button>
+      <a href={t.faq.catalogPath}>
+        <button className="mt-[100px] h-[100px] w-[250px] bg-cerise rounded-[40px] border-cerise flex items-center justify-center px-[30px] flex-col">
```

Comment:
Maybe we should change this to simply `Product Catalog` since they are no longer download it?

Reply:
Fixed